docs/BUILD

```diff
@@ -9,20 +9,24 @@ exports_files(["protodoc_manifest.yaml"])
 
 envoy_package()
 
-# TODO(phlax): fix failing/excluded configs
-# the following config only fails on windows:
-#       dns-cache-circuit-breaker: "Error: unable to read file: /etc/ssl/certs/ca-certificates.crt"
-
 filegroup(
     name = "configs",
     srcs = glob(
         ["root/**/*.yaml"],
         exclude = [
+            # `some.customer.filter`
             "root/intro/_include/life-of-a-request.yaml",
+            # "Error: No TLS certificates found for server context"
```

Comment:
ive managed to clean this file up quite a bit, but i couldnt validate it - with above error remaining

Reply:
Can we generate fake self signed certs to make this pass? (Similar to certs.sh in the integration tests)

Reply:
it is using those certs, and i added the required `.der` file. Not sure why it still failed - something not right with either certs or config - i guess i can pull all of the parts out and experiment locally

Reply:
config fixed. Im not 100% clear if the way i have changed it in terms of certs in entirely correct